app/__init__.py

```diff
@@ -0,0 +1,42 @@
+from flask import Flask
+from flask_compress import Compress
+from flask_talisman import Talisman
+from jinja2 import ChoiceLoader, PackageLoader, PrefixLoader
+
+app = Flask(__name__, static_url_path="/assets")
+
+app.jinja_loader = ChoiceLoader(
+    [
+        PackageLoader("app"),
+        PrefixLoader({"govuk_frontend_jinja": PackageLoader("govuk_frontend_jinja")}),
+    ]
+)
+
+app.jinja_env.trim_blocks = True
+app.jinja_env.lstrip_blocks = True
+
+csp = {
```

Comment:
should this live here?

Reply:
Moving this stuff into a function which is called in the __init__.py, to make it more explicit (and pythonic) to the developer team.